docs/development/release.md

```diff
@@ -6,13 +6,59 @@ We use a global versioning approach, meaning version numbers in cumulus are cons
 
 Read more about the semantic versioning [here](https://docs.npmjs.com/getting-started/semantic-versioning).
 
+## Pre-release testing
+
+Before releasing a new major version of Cumulus that contains breaking changes, we should test the deployment upgrade from the latest stable release of Cumulus to the upcoming release with breaking changes.
+
+It is preferable to use the [`cumulus-template-deploy`](`https://github.com/nasa/cumulus-template-deploy`) repo for testing the deployment, since this is the officially recommended deployment configuration for the users.
+
+1. Checkout the `cumulus-template-deploy` repo
+2. Update the deployment code to use the latest stable release artifacts if it hasn't already been updated:
+
+    ```text
+    # for data-persistence
+    source = "https://github.com/nasa/cumulus/releases/download/v5.0.1/terraform-aws-cumulus.zip//tf-modules/data-persistence"
```

Comment:
```suggestion
    source = "https://github.com/nasa/cumulus/releases/download/v5.0.1/terraform-aws-cumulus.zip/tf-modules/data-persistence"
```

Reply:
It's weird, but in the case of the release artifacts, the `//` is actually correct. The reason why is that the full `terraform-aws-cumulus.zip` contains all of our Terraform modules so that relative paths between them are supported (for example https://github.com/nasa/cumulus/blob/master/tf-modules/cumulus/archive.tf#L2). So we have one zipfile for both data-persistence and cumulus and you have to reference them directly in the zipfile using this syntax.

See https://www.terraform.io/docs/language/modules/sources.html#modules-in-package-sub-directories

Reply:
Oh interesting!